scheduleapi.py

```diff
@@ -0,0 +1,21 @@
+import json
+from flask import request
+from flask_restful import Resource
+
+from schedule import Schedule
+
+def read_args(request):
+    return json.loads(request.data.decode('utf-8'))
+
+class ScheduleAPI(Resource):
+    # Return entire schedule with details info on events
+    def get(self, owner):
+        sched = Schedule.get(owner)
```

Comment:
all validations should be done within the app class

Reply:
But this is the app class